estate/models/estate_property.py

```diff
@@ -3,17 +3,15 @@
 
 
 def _default_date_availability(self):
-        return fields.Date.context_today(self) + relativedelta(months=3)
+        return fields.Date.context_today(self) 
```

Comment:
Pourquoi ce changement ?

Reply:
C'est juste pour changer la date par DEFAUT; mais je peux retourner l'ancien si NECESSAIRE
```suggestion
        return fields.Date.context_today(self) + relativedelta(months=3)
```